docs/user-guides/devcontainers/index.md

```diff
@@ -1,87 +1,137 @@
 # Dev Containers Integration
 
-The Dev Containers integration enables seamless creation and management of Dev
-Containers in Coder workspaces. This feature leverages the
+The Dev Containers integration enables seamless creation and management of dev
+containers in Coder workspaces. This feature leverages the
 [`@devcontainers/cli`](https://github.com/devcontainers/cli) and
 [Docker](https://www.docker.com) to provide a streamlined development
 experience.
 
-This implementation is different from the existing
-[Envbuilder-based Dev Containers](../../admin/templates/managing-templates/devcontainers/index.md)
-offering.
-
 ## Prerequisites
 
 - Coder version 2.24.0 or later
-- Coder CLI version 2.24.0 or later
-- **Linux or macOS workspace**, Dev Containers are not supported on Windows
-- A template with:
-  - Dev Containers integration enabled
-  - A Docker-compatible workspace image
-- Appropriate permissions to execute Docker commands inside your workspace
+- Docker available inside your workspace
+- The `@devcontainers/cli` installed in your workspace
+
+Dev Containers integration is enabled by default. Your workspace needs Docker
+(via Docker-in-Docker or a mounted socket) and the devcontainers CLI. Most
+templates with Dev Containers support include both—see
+[Configure a template for dev containers](../../admin/templates/extending-templates/devcontainers.md)
+for setup details.
+
+## Features
+
+- Automatic dev container detection from repositories
+- Seamless container startup during workspace initialization
+- Change detection with dirty state indicators
+- On-demand container rebuild via dashboard button
+- Integrated IDE experience with VS Code
+- Direct SSH access to containers
+- Automatic port detection
+
+## Getting started
+
+### 1. Add a devcontainer.json
```

Comment:
nit: instead of ### just **1. Add a devcontainer.json** (bold)?

Reply:
My preference would be to drop the `1.` instead as the `**` would become load-bearing (if you remove it, it becomes an actual bullet list resulting in messed up numbering).